src/ploneorg/addonlisting/browser/templates/addonfolder_view.pt

```diff
@@ -0,0 +1,45 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      i18n:domain="ploneorg.addonlisting"
+      metal:use-macro="context/main_template/macros/master">
+<body>
+  <metal:content-core fill-slot="content-core">
+
+  <metal:addonlist define-macro="addonlist">
```

Comment:
I am not sure if you need a macro here, do you want to reuse this section in another template?